### PR TITLE
Edits to Astropy Units, Quantities, and Constants

### PR DESCRIPTION
**Pull request by pllim**
Changes to the "Astropy Units, Quantities, and Constants" section include but not limited to:

* Adding a table of contents to show you that there is more materials that 30 minutes can possibly cover, especially if including the PDF slides (9 pages) beforehand.
* Added new materials for constants science state.
* Completely removed the "little h" example at the end, as we won't have time for it anyway.
* Clarified the formula for Exercise 3 Part 2. (This is an Astropy workshop, not an astronomy test.)
* Grammatical fixes.
* Minor code clean-ups.

### CODE REVIEW

File: 03-UnitsQuantities/Astropy_Units.ipynb [hunk 1]
```diff
@@ -1345,7 +1437,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# answer here (Sun)"
+    "# Answer here (jWST and Sun)"
```
Comment (larrybradley) [inline 03-UnitsQuantities/Astropy_Units.ipynb]: JWST

**Comment by pllim** [comment]
@larrybradley , I am not familiar with the `<<` operator, can you please elaborate on how it can be added? Thanks.

**Comment by kelle** [comment]
Relevant tweet about >>

https://twitter.com/emilydoesastro/status/1202166518156603392?s=20

“Astro coding tip: I just discovered the bitwise operator "<<" and that it
can speed up use of the astropy.units module by a factor of over 2000 (!!!).
Tl;dr: multiplying a big array by a unit (e.g. my_array * u.deg) has to
make a copy of the array, but my_array << u.deg doesn't!”
-- 
kelle (via iOS)

File: 03-UnitsQuantities/Astropy_Units.ipynb (hunk 1 above)
Comment (larrybradley) [reply 03-UnitsQuantities/Astropy_Units.ipynb]:
```suggestion
    "# Answer here (JWST and Sun)"
```

**Comment by larrybradley** [comment]
@pllim Exactly what @kelle said.  There's a good description of the `<<` operator in the `Units` performance tips: 
 https://docs.astropy.org/en/stable/units/index.html#performance-tips

**Comment by pllim** [comment]
@larrybradley and @kelle , did I sufficiently address the comment about `<<` operator? Thanks for the info!

**Comment by larrybradley** [comment]
@pllim Are you using `astropy 4.0.x` for this demo???  I forgot to mention that `astropy.physical_constants.get()` and `astropy.astronomical_constants.get()` do not work with v3.2.3.  The section about constants not be constant may need to be removed....

**Comment by pllim** [comment]
> Are you using `astropy 4.0.x` for this demo???

Yes? At least until we have a definite decision from #80

**Comment by pllim** [comment]
Even with 3.2, there was a `constants` context manager. But we deprecated that in 4.0.
